lib/astarte_vmq_plugin/config.ex

```diff
@@ -111,7 +144,6 @@ defmodule Astarte.VMQ.Plugin.Config do
       {:username, value} -> {:username, to_string(value)}
       {:password, value} -> {:password, to_string(value)}
       {:virtual_host, value} -> {:virtual_host, to_string(value)}
-      {:host, value} -> {:host, to_string(value)}
```

Comment:
Hmm (the previous comment meant "remove the edit", not "remove the line")

Reply:
Actually it matches in the catchall in the following line (`other -> other`)

Reply:
But it doesn't convert the host to string